wolfcrypt/src/wc_port.c

```diff
@@ -1177,6 +1177,21 @@ int wc_strncasecmp(const char *s1, const char *s2, size_t n)
 }
 #endif /* USE_WOLF_STRNCASECMP */
 
+char* wc_strdup(const char *src, int memType) {
```

Comment:
This should not be always exposed... It needs to be conditional based on `#ifdef XGETENV` or possibly another macro like `USE_WOLF_STRDUP`. I'd like to see this strdup function perfectly match the stdlib strdup API. Also consider fixing `#define BUF_strdup strdup` to use it...

Reply:
Done